lint/linter.py

```diff
@@ -1092,7 +1092,7 @@ def process_match(self, m, vv):
             "linter": self.name,
             "error_type": error_type,
             "code": m.error or m.warning or '',
-            "msg": m.message,
+            "msg": m.message.strip(),
```

Comment:
I thought about this today, and looked at different code spots where we use this. 

I think this is okay (as I said I looked today). But, I asked you to just file an issue about this. The change is simple, and you could have just let this rstrip here bc right now we don't have a need for this anywhere in the code. And with an issue open everyone gets involved and can raise a hand. 

As I said again, **I** think this is safe and reasonable, and I would have raised my hand in such a issue, but the other contributors have to have a look here as well.

Reply:
Sorry. Missed that.

Reply:
Sure, I have nothing against strip here. 

We do a `line.rstrip()` elsewhere... perhaps we should sanitise our values all in one place so everything else can just rely on it being stripped?

Reply:
Absolutely! I have opened an issue for it #854.
My suggestion let's just leave the `strip` there, because it is already moved out of `panel.py`
and we later on centralize sanitation.

Reply:
Works for me